Dockerfile.template

```diff
@@ -55,13 +55,18 @@ RUN set -ex; \
 	; \
 	rm -rf /var/lib/apt/lists/*
 
+# Allowing overriding of REPOSITORY / REPOSITORY_KEY for testing
+ARG REPOSITORY
+ARG REPOSITORY_KEY
```

Comment:
At least `ARG REPOSITORY` could probably be moved a bit further down for better cache (if you care about that for these testing bulids).